projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -339,15 +339,17 @@ public function add_plugins_menu( $wp_admin = false ) {
 	 * @param bool $wp_admin Optional. Whether links should point to Calypso or wp-admin. Default false (Calypso).
 	 */
 	public function add_users_menu( $wp_admin = false ) {
+		$submenus_to_update = array();
 		if ( current_user_can( 'list_users' ) ) {
 			if ( ! $wp_admin ) {
 				$submenus_to_update = array(
-					'users.php'    => 'https://wordpress.com/people/team/' . $this->domain,
 					'user-new.php' => 'https://wordpress.com/people/new/' . $this->domain,
 					'profile.php'  => 'https://wordpress.com/me',
 				);
-				$this->update_submenus( 'users.php', $submenus_to_update );
 			}
+			// We shall add the Calypso User management screen at all cases ( Calypso & Atomic ).
+			$submenus_to_update = array_merge( $submenus_to_update, array( 'users.php' => 'https://wordpress.com/people/team/' . $this->domain ) );
```

Comment:
Since folks still have access to `wp-admin/user-new.php` from the "Advanced user management" screen that we are now always sharing, what if we also always update the "Add new" menu with the Calypso link? That should avoid most of the cases where an admin incorrectly creates a local user.

Reply:
Good point! Fixed here 866d56f